analysis/wwz/wwz4l.py

```diff
@@ -206,9 +208,13 @@ def process(self, events):
             sow_factDown       = -1
 
 
-        datasets = ["SingleMuon", "SingleElectron", "EGamma", "MuonEG", "DoubleMuon", "DoubleElectron", "DoubleEG"]
+        datasets = ["SingleMuon", "SingleElectron", "EGamma", "MuonEG", "DoubleMuon", "DoubleElectron", "DoubleEG","Muon"]
         for d in datasets:
-            if d in dataset: dataset = dataset.split('_')[0]
+            if d in dataset:
```

Comment:
Could you explain what this is doing?

Reply:
It seems the dataset split thing is still happening outside the if block. I would put all of this stuff inside the if since it's only relevant for data. Also, one comment describing what it's doing would be useful.